pom.xml

```diff
@@ -247,12 +250,6 @@
 									</resources>
 								</configuration>
 							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<artifactId>maven-resources-plugin</artifactId>
```

Comment:
Why is the resources plugin being removed?

Reply:
Two executions that use same plugin are now combined. Avoids maven warnings.